tests/conftest.py

```diff
@@ -1300,6 +1300,9 @@ def additional_testsuite_properties(record_testsuite_property, pytestconfig):
     launch_url = config.REPORTING.get("rp_launch_url")
     if launch_url:
         record_testsuite_property("rp_launch_url", launch_url)
+    # add merkers as separated property
```

Comment:
```suggestion
    # add markers as separated property
```

Reply:
Fixed in [833cfd0](https://github.com/red-hat-storage/ocs-ci/pull/5605/commits/833cfd0efe44117b18862f0563321a0cd6b496a8).